app/models/datatables/datatables_team.rb

```diff
@@ -0,0 +1,14 @@
+module Datatables
```

Comment:
I don't know what I have seen people treat this as an equivalent to active model.

Reply:
Well it's up to us really. I think it's a bit confusing that it's not clear from this file that it is describing a view, rather than a database table. I think it would make sense to have it separated into its own folder, just to avoid confusion (e.g. a junior developer wanting to add a column to the `DatatablesTeam`, and struggling why the database migration doesn't work).